adapters/adform/adform_test.go

```diff
@@ -374,6 +377,9 @@ func createOpenRtbRequest(testData *aBidInfo) *openrtb.BidRequest {
 	bidRequest.Regs = &regs
 	bidRequest.User.Ext = getUserExt()
 
+	bidRequest.Cur = make([]string, 1)
```

Comment:
Just food for thought - You can also combine these lines to achieve the same: `bidRequest.Cur = []string{testData.currency}`

Reply:
We'll skip it for now to not rebuild and retest for a minor change